tutorials/rendering/jitter_stutter.rst

```diff
@@ -241,6 +459,127 @@ If your mouse offers multiple :abbr:`DPI (Dots Per Inch)` settings, consider als
+
+This table is read like this:
+ 
+ - If Waitable Swapchains are available, at "low" Godot will target 1 frame of latency. Otherwise, at "low" it will use AUTO as fallback.
+ - If Waitable Swapchains are available, at "medium" Godot will use the value in frame_queue_size  as target for frames of latency. Otherwise it uses PARALLEL mode (which is the same as having no pacing method).
+
+
+.. tip::
+
+    Godot's behavior on version 4.4 and earlier corresponds to ``high_throughput``.
+
+**Troubleshooting:**
+
+If you suspect Godot's pacing methods are malfunctioning, there are several steps you can take.
+
+The first thing we need to do is to run with ``--verbose`` to know which pacing methods are available.
+You should see something like this:
+
+.. code:: text
+
+    Supported Pacing Methods (mask 03):
+        SEQUENTIAL_SYNC
+        WAITABLE_SWAPCHAIN
+    Current Pacing Method (may change later): WAITABLE_SWAPCHAIN
+
+In this case, Waitable Swapchain is being used. This is great! The second thing we should try is running with a different latency mode:
+Let's say: ``--latency-mode medium``. There's 4 options to try (note: ``high_throughput`` just disables any pacing method).
+If the problem is gone, then we're done.
+
+But let's say we want to try another method. For some reason, we suspect waitable swapchains are malfunctioning.
+We can try masking this feature out:
+
+- ``--pacing-mode-mask 0x01`` will force to only use SEQUENTIAL_SYNC (if available).
+- ``--pacing-mode-mask 0x02`` will force to only use WAITABLE_SWAPCHAIN (if available).
+
+The following table shows the masks for each settings (these masks can be OR'ed together):
+
+.. table::
+   :widths: auto
+
+   +--------------------+------+
+   | Setting            | Mask |
+   +====================+======+
+   | SEQUENTIAL_SYNC    | 0x01 |
+   +--------------------+------+
+   | WAITABLE_SWAPCHAIN | 0x02 |
+   +--------------------+------+
+   | ANDROID_SWAPPY     | 0x04 |
+   +--------------------+------+
+
+Thus by launching with ``--pacing-mode-mask 0x01`` we can experience how it runs by using ``SEQUENTIAL_SYNC``.
+Now that we are in sequential sync pacing mode; again we can try ``--latency-mode`` makes any difference.
+You can try ``low_extreme`` (SEQUENTIAL), ``low`` (AUTO) and ``medium`` (PARALLEL) to see if they make a difference.
+
+**When to report issues:**
+
+ - If WAITABLE_SWAPCHAIN at ``low`` or ``medium`` settings are causing problems. This could indicate an issue with the rendering API, the OS, or HW.
+ - If SEQUENTIAL_SYNC at ``low`` is causing problems but works fine at ``low_extreme`` or ``medium``. This would indicate a problem in Godot's AUTO algorithm.
+ - If ANDROID_SWAPPY is causing problems.
+
+**What NOT to report:**
+ - ``low`` or ``low_extreme`` reduces the FPS.
+ - ``AUTO`` does not improve latency.
+ - ``SEQUENTIAL_SYNC`` does not improve latency.
```

Comment:
```suggestion
**When to report issues:**

 - If ``WAITABLE_SWAPCHAIN`` at ``low`` or ``medium`` settings are causing problems. This could indicate an issue with the rendering API, the OS, or HW.
 - If ``SEQUENTIAL_SYNC`` at ``low`` is causing problems but works fine at ``low_extreme`` or ``medium``. This would indicate a problem in Godot's AUTO algorithm.
 - If ``ANDROID_SWAPPY`` is causing problems.

**What NOT to report:**
 - ``low`` or ``low_extreme`` reduces the FPS.
 - ``AUTO`` does not improve latency.
 - ``SEQUENTIAL_SYNC`` does not improve latency.
```